components/brave_shields/browser/brave_shields_util_unittest.cc

```diff
@@ -740,152 +740,6 @@ TEST_F(BraveShieldsUtilTest, SetFingerprintingControlType_ForOrigin) {
   EXPECT_EQ(ControlType::DEFAULT, type);
 }
 
-/* HTTPSEVERYWHERE CONTROL */
-TEST_F(BraveShieldsUtilTest, SetHTTPSEverywhereEnabled_Default) {
-  auto* map = HostContentSettingsMapFactory::GetForProfile(profile());
-  // settings should be default
-  auto setting = map->GetContentSetting(
-      GURL(), GURL(), ContentSettingsType::BRAVE_HTTP_UPGRADABLE_RESOURCES);
```

Comment:
Did you remove this content setting type? Maybe I missed it somewhere in here. We should also eventually delete all the existing settings. We don't want to do it right away because we don't want them to sync to devices that may be an older versions, but usually after one year I think we'd clear them.

Reply:
I didn't remove it yet. Makes sense to hold on for a while.

Reply:
why would we hold onto it if we're not using it?